pkg/network/usm/ebpf_ssl.go

```diff
@@ -611,6 +612,19 @@ func addHooks(m *manager.Manager, procRoot string, probes []manager.ProbesSelect
 		}
 		defer elfFile.Close()
 
+		arch, err := bininspect.GetArchitecture(elfFile)
+		if err != nil {
+			return err
+		}
```

Comment:
1. let's add the same for go-tls
2. let's try to add a UT to verify it is prevented

Reply:
1. go-tls doesn't need this, since when the application itself is in a different architecture it cannot be executed directly, but only via a wrapper such as qemu, and in that case the `exe` will point to `qemu` and not to the application itself. (go-tls already rejects 32-bit applications running on 64-bit)
2. Done.